src/test/java/hudson/plugins/copyartifact/TriggeredBuildSelectorTest.java

```diff
@@ -551,6 +576,9 @@ public void testBackwardCompatibility() throws Exception {
         d.setGlobalUpstreamFilterStrategy(TriggeredBuildSelector.UpstreamFilterStrategy.UseOldest);
         
         FreeStyleProject upstream = j.createFreeStyleProject();
+        ParameterDefinition paramDef = new StringParameterDefinition("CONTENT","foo");
```

Comment:
`ParameterDefinition paramDef = new StringParameterDefinition("CONTENT", "foo");`

In this plugin, it is using this convention.